Examples/Sample_xCertificateImport_MinimalUsage.ps1

```diff
@@ -0,0 +1,14 @@
+Configuration MyNode
+{
+    Import-DscResource -ModuleName xCertificate
+
+    Node $AllNodes.NodeName
+    {
+        xCertificateImport MyTrustedRoot
+        {
+            Thumbprint = 'c81b94933420221a7ac004a90242d8b1d3e5070d'
+            Store = "Root"
+            Path = '\\Server\Share\Certificates\MyTrustedRoot.cer'
+        }
+    }
+}
```

Comment:
should end a file with a new line per contribution guidelines.  [style guidelines - general rules #5](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md#general-rules)